core/src/test/java/nl/nn/adapterframework/test/EnvironmentTest.java

```diff
@@ -0,0 +1,20 @@
+package nl.nn.adapterframework.test;
+
+import javax.xml.parsers.SAXParserFactory;
+
+import org.junit.Test;
+
+public class EnvironmentTest {
+	@Test
+	public void getJRE() {
+		String javaVersion = System.getProperty("java.version");
+		System.out.println(String.format("Java version: %s", javaVersion));
+	}
+
+	@Test
+	public void getSAXImplementation() {
+		SAXParserFactory factory = SAXParserFactory.newInstance();
+		System.out.println(String.format("SAXParserFactory implementation: %s", factory.getClass().getName()));
+	}
+}
+
```

Comment:
Er is al een J-Unit test die controleert of de juiste parsers gebruikt worden (zie TestMetaInfServices.java).
Helaas ging bij jou de build fout omdat er een classpath conflict was (2 dependencies met een andere versie).

Zou je dit bestand weg kunnen halen?